src/components/uploadDocs/UploadDocsPages.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from "react";
```

Comment:
This file now contains a parametrized version of the old `AssaysPage` component that can display either Assay or Analysis docs based on the value of the `uploadType` prop.